keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -168,11 +174,27 @@ type ControllerLeaderConfiguration struct {
 	Component string `json:"component"`
 }
 ```
+#### Default LeaderMigrationConfiguration
+
+Both `kube-controller-manager` and `cloud-controller-manager` must each provide a default `LeaderMigrationConfiguration` 
+that represents the situation where the controller manager is running with default assignments of controllers and lock type selection.
+
+The default values should be only used when no configuration file is specified. 
```

Comment:
No "should only", "will only" or "must only". (https://tools.ietf.org/html/rfc2119)

Reply:
Thanks! Updating existing "should" too.